client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx

```diff
@@ -50,8 +50,7 @@ export class DowngradeStep extends Component {
 			refundReason = (
 				<p>
 					{ translate(
-						'WordPress.com Personal still gives you access to email and chat support, ' +
-							'removal of ads, and more — and for 50% of the cost of your current plan.'
+						'WordPress.com Personal still gives you access to email support, removal of ads, and more — and for 50% of the cost of your current plan.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'WordPress.com Personal still gives you access to email and chat support, removal of ads, and more — and for 50% of the cost of your current plan.' )` *ES Score: 10*